### DISCUSSION TITLE
Shifted in Z axis

### DISCUSSION THREAD

**Discussion by xav69ier** [CBICA/CaPTk#1434] [Q&A]
Dear colleagues 
I manage successful coregistration of T1 and FLAIR to T1Gadolinium scans using https://cbica.github.io/CaPTk/preprocessing_reg.html (CLI) according to BRATS guidelines, but some of the T2 scans although also coregistered still show a certain shifting in the Z axis (perpendicular to axial plane XY). I noticed the shift because some T2 do not show the eyes of the subject but the T1 and FLAIR do. I wonder how to correct this shift. I have used so far Rigid coregistration and wonder whether I should attempt Affine. 
Many thanks in advance for any feedback: XAvier

**Comment by sarthakpati** [comment]
Since the registration depends on a lot of different factors, it is difficult to estimate which mechanism would work best... I would suggest trying different options within rigid, and if it doesn't work, to move to affine.
